src/platform/storage_winuser.c

```diff
@@ -194,7 +194,7 @@ _IRQL_requires_max_(PASSIVE_LEVEL)
 QUIC_STATUS
 CxPlatStorageReadValue(
     _In_ CXPLAT_STORAGE* Storage,
-    _In_opt_z_ const char * Name,
+    _In_z_ const char * Name,
```

Comment:
`Name` is optional

Reply:
From MSDN: https://docs.microsoft.com/en-us/windows/win32/api/winreg/nf-winreg-regqueryvalueexa

lpValueName

The name of the registry value.

If lpValueName is NULL or an empty string, "", the function retrieves the type and data for the key's unnamed or default value, if any.